src/store/modules/blocks/actions.js

```diff
@@ -228,7 +228,7 @@ export default {
 
     if (!generations.length) {
```

Comment:
Is this condition actually needed? Looks like that it is `true` only if `size` is zero.